subprojects/packagefiles/sqlpp11/meson_options.txt

```diff
@@ -0,0 +1,5 @@
+option('sqlite3_connector', type : 'boolean', value : false, description : 'Build SQLite3 connector')
```

Comment:
why not use feature options?

Reply:
I want to give the user the power to select which database connector he wants to use and not automatically enable them. It could lead to that all database connectors are enabled.

Reply:
Sounds like this should be a combo option then.

Reply:
No, not really. The user should still be able to select more then one database connector. A use-case would be transferring data from SQLite3 to PostgreSQL.